Sources/Apollo/HTTPNetworkTransport.swift

```diff
@@ -69,31 +69,38 @@ public protocol HTTPNetworkTransportRetryDelegate: HTTPNetworkTransportDelegate
 
 /// A network transport that uses HTTP POST requests to send GraphQL operations to a server, and that uses `URLSession` as the networking implementation.
 public class HTTPNetworkTransport: NetworkTransport {
+  
   let url: URL
-  let session: URLSession
+  var session: URLSession
   let serializationFormat = JSONSerializationFormat.self
-  let useGETForQueries: Bool
   let delegate: HTTPNetworkTransportDelegate?
 
+  private let _useGETForQueries: Bool
+  private let _enableAutoPersistedQueries: Bool
+  private let _useHttpGetMethodForPersistedQueries: Bool
```

Comment:
I'm trying to get us away from underscores for private properties - can you please remove those?

Reply:
updated